### PR TITLE
chore: Post release build results to slack

### PR DESCRIPTION
**Pull request by scsmithr**
Closes https://github.com/GlareDB/glaredb/issues/2431

### CODE REVIEW

File: .github/workflows/nodejs-release.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ on:
     # Run every Monday at 12:00
     - cron: 0 12 * * 1
   push:
+    branches:
+      - "sean/slack"
```
Comment (greyscaled) [inline .github/workflows/nodejs-release.yml:12]: should this be removed before merge?

Comment (scsmithr) [reply .github/workflows/nodejs-release.yml:12]: yes

**Review by greyscaled — COMMENTED** [review]
This is great

**Review by tychoish — APPROVED** [review]
is there an easy way to have there be one post per release/commit, with comments/updates about each specific release/workflow

**Review by greyscaled — APPROVED** [review]

**Comment by scsmithr** [comment]
> is there an easy way to have there be one post per release/commit, with comments/updates about each specific release/workflow

I'm sure there's a way. The slack action readme says you can do things like `update-ts: ${{ steps.slack.outputs.ts }}` but it seems that needs a reference to a previous step and I don't know of an easy of getting this "ts" across workflows.

I just went with the quickest/easiest/straightforwardest way to get us feedback on the builds. Iterating on this to have a single message sounded kinda painful. What if one of the node js jobs passed, then a python job failed? Would we be able to preserve that context in the message?

**Comment by tychoish** [comment]
> I'm sure there's a way. The slack action readme says you can do things like `update-ts: ${{ steps.slack.outputs.ts }}` but it seems that needs a reference to a previous step and I don't know of an easy of getting this "ts" across workflows.
> 
> I just went with the quickest/easiest/straightforwardest way to get us feedback on the builds. Iterating on this to have a single message sounded kinda painful. What if one of the node js jobs passed, then a python job failed? Would we be able to preserve that context in the message?

Legit! agree that it's good to merge. I think combining into one release workflow with three jobs wouldn't be too bad, and then it'd be a single workflow.

**Pull request merged**